Unix/engine/engine.c

```diff
@@ -45,7 +45,9 @@ int enginemain(int argc, const char* argv[])
     {
         err(ZT("cannot set sighandler, errno %d"), errno);
     }
-
+    
```

Comment:
do we have separate log file for engine? If not, please cross check, do we really need to handle it here?